cmd/keytransparency-sequencer/server.go

```diff
@@ -18,24 +18,26 @@ import (
 	"flag"
 	"net/http"
 
+	"github.com/google/keytransparency/cmd/serverutil"
+
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/reflection"
 
-	"github.com/google/keytransparency/cmd/serverutil"
-	ktpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
+	gpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
```

Comment:
was serverutil completely unused?

Reply:
Just moved it up a few lines to keep the following import pattern:
- System
- Key Transparency
- Third Party
- Renamed and underscore imports